### PR TITLE
feat(transport-bridge-bin): introduce binary bundle of node bridge

### PR DESCRIPTION
**Pull request by mroz22**
### Description
This PR take transport/bridge/http module added in #9991 and bundles it as binary. Once this is done we may replace bridge binaries available from https://suite.trezor.io/web/bridge/ and decrease complexity of our stack by ditching  https://github.dev/trezor/trezord-go completely. 

We don't actually want to use it in production yet. The next step now is to make it available from trezor-user-env and run all our tests against it. 

cc @Nodonisko for build process

cc I dont know who for socket-security. it looks okeyish to me, but please have another looks somebody

### Related issues
- builds on top of #9991

### CODE REVIEW

**Comment by mroz22** [comment]
@SocketSecurity ignore pkg@5.8.1
@SocketSecurity ignore pkg-fetch@3.4.2
@SocketSecurity ignore prebuild-install@7.1.1

File: packages/transport-bridge-bin/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "name": "@trezor/transport-bridge-bin",
+    "version": "1.0.0",
+    "main": "src/index",
+    "license": "See LICENSE.md in repo root",
+    "private": true,
+    "sideEffects": false,
+    "scripts": {
+        "type-check": "tsc --build",
+        "lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",
+        "build:lib": "rimraf -rf lib && yarn tsc --build ./tsconfig.json",
```
Comment (mroz22) [inline packages/transport-bridge-bin/package.json]: lets remove build:lib

File: packages/transport-bridge-bin/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "name": "@trezor/transport-bridge-bin",
```
Comment (mroz22) [inline packages/transport-bridge-bin/package.json]: transport-bridge

File: packages/transport-bridge-bin/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        "@trezor/transport": "workspace:^"
+    },
+    "pkg": {
+        "outputPath": "build",
+        "targets": [
+            "node18-macos-x64",
+            "node18-macos-arm64",
+            "node18-linux-x64",
+            "node18-linux-arm64",
+            "node18-win-x64"
```
Comment (mroz22) [inline packages/transport-bridge-bin/package.json]: lets consult signing with @vdovhanych

File: packages/transport-bridge-bin/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { TrezordNode } from '@trezor/transport/lib/bridge/http';
```
Comment (mroz22) [inline packages/transport-bridge-bin/src/index.ts]: lets move `bridge` but consider potential conflicts with szymon

File: packages/transport-bridge/package.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@trezor/transport-bridge",
+    "version": "1.0.0",
```
Comment (tsusanka) [inline packages/transport-bridge/package.json]: Total nit, which I am sure is too soon for, but it'd be great to version the node version of bridge as v3. That means 3.0.0. for users.

File: packages/transport-bridge/package.json [hunk 6]
```diff
@@ -1,19 +1,39 @@
 {
     "name": "@trezor/transport-bridge",
-    "version": "1.0.0",
+    "version": "3.0.0",
```
Comment (marekrjpolak) [inline packages/transport-bridge/package.json:3]: Only odd versions are allowed? :)

**Edit**: Oooh, it's probably this, right? https://github.com/trezor/trezor-suite/pull/10356#discussion_r1447586576

File: packages/transport-bridge/README.md [hunk 7]
```diff
@@ -1,3 +1,13 @@
 # @trezor/transport-bridge
 
 -   javascript clone of https://github.com/trezor/trezord-go
+
+## Build and run module
+
+-   `yarn workspace @trezor/transport-bridge build:js`
+-   `node ./packages/transport-bridge/lib/bin.js`
```
Comment (marekrjpolak) [inline packages/transport-bridge/README.md]: There is no `lib` folder! :smiling_imp: 
```suggestion
-   `node ./packages/transport-bridge/dist/bin.js`
```

**Review by marekrjpolak — APPROVED** [review]

File: packages/transport-bridge/README.md (hunk 7 above)
Comment (mroz22) [reply packages/transport-bridge/README.md]: oh yea, good catch

File: packages/transport-bridge/package.json (hunk 6 above)
Comment (mroz22) [reply packages/transport-bridge/package.json:3]: yes, this is intentional

**Pull request merged**